InvenTree/config_template.yaml

```diff
@@ -154,6 +154,11 @@ static_root: '/home/inventree/data/static'
 # Use environment variable INVENTREE_LOGIN_ATTEMPTS
 #login_attempts: 5
 
+# Add new user on first startup
```

Comment:
Should these be called `admin_user` / `admin_email` / `admin_password` for improved clarity?